Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/administration/TestEditPage.tsx

```diff
@@ -0,0 +1,17 @@
+import { useParams } from 'react-router';
+import isNil from 'lodash/isNil';
+
+import { RedirectExternal } from '../../components/common/RedirectExternal';
+import { getAdministrationTestEditUrl } from '../../utils/urls';
+
+const TestEditPage = () => {
+    const params = useParams();
+    const { testId } = params;
+    if (isNil(testId)) {
+        return null;
+    }
+
+    return RedirectExternal(getAdministrationTestEditUrl(testId));
+};
```

Comment:
What is the purpose of this page?

Reply:
To redirect me to the external page. If I try and redirect directly from the submission details, then I get an error.